docs/docs/guides/migrate/users.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Migrate Users
+sidebar_label: Users
+---
+
+Migrating users from an existing system, while minimizing impact on said users, can be a challenging task.
+
+## Individual Users
+
+Creating individual users can be done with this endpoint: [ImportHumanUser](/docs/apis/mgmt/management-service-import-human-user).
+Please also consult our [guide](/docs/guides/manage/user/reg-create-user) on how to create users.
+
+```json
+{
+  "userName": "test9@test9",
+  "profile": {
+    "firstName": "Road",
+    "lastName": "Runner",
+    "displayName": "Road Runner",
+    "preferredLanguage": "en"
+  },
+  "email": {
+    "email": "test@test.com",
+    "isEmailVerified": false
+  },
+  "hashedPassword": {
+    "value": "$2a$14$aPbwhMVJSVrRRW2NoM/5.esSJO6o/EIGzGxWiM5SAEZlGqCsr9DAK",
+    "algorithm": "bcrypt"
+  },
+  "passwordChangeRequired": false,
+  "otpCode": "testotp",
+  "requestPasswordlessRegistration": false,
+  "idps": [
+    {
+      "configId": "124425861423228496",
+      "externalUserId": "roadrunner@mailonline.com",
+      "displayName": "name"
+    }
+  ]
+}
+```
+
+## Bulk import
+
+For bulk import use the [import endpoint](https://zitadel.com/docs/apis/admin/admin-service-import-data) on the admin API:
+
+```json
```

Comment:
I don't mind, but not sure if we want to include my personal data in here.

Reply:
replaced